pkg/builder/helmAppService.go

```diff
@@ -101,6 +106,20 @@ func (impl *ApplicationServiceServerImpl) GetDeploymentDetail(ctx context.Contex
 	return GetDeploymentDetail(in)
 }
 
+func (impl *ApplicationServiceServerImpl) InstallRelease(ctx context.Context, in *client.InstallReleaseRequest) (*client.InstallReleaseResponse, error) {
+	impl.logger.Infow("install release request")
```

Comment:
add some contextual info in log

Reply:
Will do this in my next item (Refactoring #1152)